prime-angular-frontend/src/app/lib/validators/form-control.validators.ts

```diff
@@ -62,7 +62,7 @@ export class FormControlValidators {
    */
   public static email(control: AbstractControl): ValidationErrors | null {
     if (!control.value) { return null; }
-    const regExp = /^[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$/i;
+    const regExp = /^[a-z0-9._%'+-]+@[a-z0-9.-]+\.[a-z]{2,}$/i;
```

Comment:
I don't know if this Wikipedia article is definitive but even for unquoted local-parts of the email address, a lot of atypical characters are allowed **!#$%&'*+-/=?^_`{|}~** 😱  https://en.wikipedia.org/wiki/Email_address  Should they all be allowed?

Reply:
Note that @itchison is checking with CHES team 
![image](https://user-images.githubusercontent.com/7586665/158477210-7e761748-92d4-412c-b622-ad9f134a6a57.png)

Reply:
> Note that @itchison is checking with CHES team ![image](https://user-images.githubusercontent.com/7586665/158477210-7e761748-92d4-412c-b622-ad9f134a6a57.png)

[CHES](https://github.com/bcgov/common-hosted-email-service) uses [validatorjs](https://github.com/validatorjs/validator.js) which allows single quote in emails.